src/Symfony/Component/Console/Tests/ApplicationTest.php

```diff
@@ -1115,6 +1116,13 @@ public function testCanCheckIfTerminalIsInteractive()
         $inputStream = $tester->getInput()->getStream();
         $this->assertEquals($tester->getInput()->isInteractive(), @posix_isatty($inputStream));
     }
+
+    public function testAddApplicationReturnsNullWhenCommandIsNotEnabled()
+    {
+        $application = new CustomDefaultCommandApplication();
+
+        $this->assertNull($application->add(new \DisabledCommand()));
```

Comment:
>Here $application is always new object so it will null and you assert it should be null. I think it is not helpful testcase.

No the assert is on the return value of the call to `add()`, which passes while it shouldn't.
The PHPDoc states it will always return the command:
https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Console/Application.php#L379

but it doesn't:
https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Console/Application.php#L388

I've opened a PR to correct it; https://github.com/symfony/symfony/pull/19389